src/main/java/com/hivemq/cli/ioc/ShellModule.java

```diff
@@ -69,6 +69,7 @@
                 .addSubcommand(shellExitCommand)
                 .setColorScheme(config.getColorScheme())
                 .setUsageHelpWidth(config.getCliWidth())
+                .setCaseInsensitiveEnumValuesAllowed(true)
```

Comment:
I think you wanted this to be set to the SwamModule?

Reply:
Is there any reason why other commands should be case sensible?

Reply:
No, I think we just almost never used enums until now.